include/extractor/guidance/turn_instruction.hpp

```diff
@@ -42,8 +42,16 @@ enum TurnType // at the moment we can support 32 turn types, without increasing
     NewName,                // no turn, but name changes
     Continue,               // remain on a street
     Turn,                   // basic turn
+    FirstTurn,              // First of x turns
+    SecondTurn,             // Second of x turns
+    ThirdTurn,              // Third of x turns
+    FourthTurn,             // Fourth of x turns
```

Comment:
because this is the only thing that happens in not fully messed up data-modelling. 
Haven;t seen anything that has more than four turns on a single side and wasn't a data modelling problem. They will probably make room for something better down the line.